### DISCUSSION TITLE
Keycloak admin redirection url not working

### DISCUSSION THREAD

**Discussion by caregoigi** [keycloak/keycloak#34628] [Archive]
Hi,

when open the admin console it redirect to another url and show me 
We are sorry...
Invalid parameter: redirect_uri
![Screenshot from 2024-11-04 19-24-00](https://github.com/user-attachments/assets/f6416af9-55d1-426a-9227-0d1f37775e65)

**Comment by adropofliquid** [comment]
Hi @caregoigi can you provide more information about this? where are you accessing the admin console from?

**Reply by caregoigi** [reply]
After changing the password, we hit the logout button. After that, this appears as it does in the screenshot above.  I am unable to access Keycloak Dashborad.

**Reply by caregoigi** [reply]
We have deployed this on kubernet ecr and using the loadBalancer external id. 
This is admin console url: http://a3f52700d90c94daf9161fa689014fb8-1869986689.eu-central-1.elb.amazonaws.com:8080/
After hiting it will redirect it to 
http://a3f52700d90c94daf9161fa689014fb8-1869986689.eu-central-1.elb.amazonaws.com:8080/realms/master/protocol/openid-connect/auth?client_id=security-admin-console&redirect_uri=http%3A%2F%2Fa3f52700d90c94daf9161fa689014fb8-1869986689.eu-central-1.elb.amazonaws.com%3A8080%2Fadmin%2Fmaster%2Fconsole%2F&state=e30d35fd-ca16-4f42-8f71-df71ac5072a8&response_mode=query&response_type=code&scope=openid&nonce=512f0224-580c-426a-9ccc-dcf8e9a326ec&prompt=none&code_challenge=9ANqILULmH2q2nJ5R7GR_aWQVNDlOg1A3VqN6VBZayQ&code_challenge_method=S256
Where we show the We are sorry...
Invalid parameter: redirect_uri

[« Back to Application](http://a3f52700d90c94daf9161fa689014fb8-1869986689.eu-central-1.elb.amazonaws.com:8080/admin/master/console/)

How can we fixed it without login?
